res/res_respoke/respoke_endpoint.c

```diff
@@ -43,6 +43,60 @@
 #include "include/respoke_app.h"
 #include "include/respoke_sdk_header.h"
 
+#define DEFAULT_STATE_BUCKETS 53
+
+static AO2_GLOBAL_OBJ_STATIC(endpoint_states);
```

Comment:
If being completely honest, that would be because that's how res_pjsip_outbound_registration handles it.

Arguably, you do have potential race conditions acting on the state container, wherein an ARI request comes in, asks for the state container, while at the same time we are shutting down. Kerblooey.